mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/options.scala

```diff
@@ -203,6 +203,22 @@ object Config {
     def temporalPropsOrDefault = temporalProps.getOrElse(List.empty)
   }
 
+  /**
+   * Configuration of trace evaluation
+   *
+   * @param trace
+   *   an ITF trace file
+   * @param expressions
+   *   a list of expressions, to be evaluated at every state in the trace
+   */
+  case class Tracee(
+      trace: Option[File] = None,
```

Comment:
I think this should take a `SourceOption`, just as we do for the `Input` configuration:

https://github.com/informalsystems/apalache/blob/e03bc42258aa844869794bf5efc7bb440628db93/mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/options.scala#L118-L127

This will allow us to support RPCs and other modes of invocation.